Corona-Warn-App/src/deviceForTesters/java/de/rki/coronawarnapp/test/keydownload/ui/CachedKeyListItem.kt

```diff
@@ -0,0 +1,12 @@
+package de.rki.coronawarnapp.test.keydownload.ui
+
+import de.rki.coronawarnapp.diagnosiskeys.storage.CachedKeyInfo
+import de.rki.coronawarnapp.util.lists.HasStableId
+
+data class CachedKeyListItem(
+    val info: CachedKeyInfo,
+    val fileSize: Long
+) : HasStableId {
+    override val stableId: Long
```

Comment:
Minor: how about  introducing a typealias for stableId so we don't pass Long params araound

Reply:
Don't see the benefit here though :thinking: ? The stableID is a long, hiding that from the developer, and somehow obscuring the relation to recyclerview adapter's "stableID" seems to be disadvantageous.

This is not a custom property we invented. https://developer.android.com/reference/androidx/recyclerview/widget/RecyclerView.Adapter#hasStableIds()